packages/eslint-plugin/src/util/escapeRegExp.ts

```diff
@@ -0,0 +1,8 @@
+const reRegExpChar = /[\\^$.*+?()[\]{}|]/g;
+const reHasRegExpChar = RegExp(reRegExpChar.source);
+
+export function escapeRegExp(string = ''): string {
+  return string && reHasRegExpChar.test(string)
+    ? string.replace(reRegExpChar, '\\$&')
+    : string;
+}
```

Comment:
you can't just copy paste code from other OSS projects without respecting their licence.
At the very least you should link back to lodash's licence

Reply:
Rebased and license & credit has been added.